README.md

```diff
@@ -166,6 +166,21 @@ end
 Calculator.start_link(x, name: :foo)
 ```
 
+This will allow you to pass in the actor process name when instantiated as a worker by the application like so:
```

Comment:
Ah, makes sense. For me, any example that has the `worker` keyword works :) I'll make this change and squash the commits in a bit